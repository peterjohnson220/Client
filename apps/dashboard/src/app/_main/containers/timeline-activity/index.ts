export * from './timeline-activity.component';
export * from './timeline-panel.component';
