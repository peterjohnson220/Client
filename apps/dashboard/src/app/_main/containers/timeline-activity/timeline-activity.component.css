.avatar{
  width: 30px;
  height: 30px;
  border-radius: 30px;
  box-shadow: 0 2px 8px rgba(0, 0, 0, .3);
}

.h6{
  display: inline;
}

.fa-circle-thinner:before {
  box-sizing: border-box;
  display: block;
  width: 2em;
  height: 2em;
  padding-top: 0.3em;
  text-align: center;
  border-radius: 100%;
  border: 2px solid #d9d9d9;
}

.timeline{
  list-style: none;
  padding:0 0 20px;
  position:relative;
  margin-top:-15px
}

.timeline-body{
  background-color: #f5f5f6;
  border-radius: 5%;
  border: thin;
  border-color:  #f5f5f6;
}

.timeline-icon{
  background-color: #ffffff;
}

.timeline:before{
  top:30px;
  bottom:25px;
  position:absolute;
  content:" ";
  width:3px;
  background-color:#eeeeee;
  left:25px;
  margin-right:-1.5px
}
.timeline>li,.timeline>li>.timeline-panel{
  margin-bottom:5px;
  position:relative
}

.timeline>li:after,.timeline>li:before{
  content:" ";
  display:table
}
.timeline>li:after{
  clear:both
}

.timeline>li>.timeline-panel{
  margin-left:55px;
  float:left;
  top:19px;
  padding:4px 10px 8px 15px;
}

.timeline>li>.timeline-icon{
  width:36px;
  height:36px;
  position:absolute;
  top:26px;left:9px;
  margin-right:-25px;
  background-color:#fff;
  z-index:100;
}

