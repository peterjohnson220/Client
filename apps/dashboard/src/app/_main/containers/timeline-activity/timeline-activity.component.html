<div class="card p-0 timeline-container">
  <div class="card-header d-flex align-items-center pl-1">
    <i class="far fa-3x fa-comment-alt px-2 ml-1" aria-hidden="true"></i>
    <h4>Recent Activity</h4>
    <button (click)="toggleFilterMenu()" class="reset-button px-10">
      <div *ngIf="showFilter">
        <i class="fa fa-2x px-1 fa-sliders-h filter-icon-on" aria-hidden="true"></i>
      </div>
      <div *ngIf="!showFilter">
        <i class="fa fa-2x px-1 fa-sliders-h filter-icon-off" aria-hidden="true"></i>
      </div>
    </button>
  </div>
  <div>
    <div class="timeline-filter m-0 p-0 pt-3"
         [ngClass]="{'hide': !showFilter, 'show': showFilter }"
         *ngIf="timelineActivityFilters.length > 0">
      <ul>
          <li class="py-1 d-flex" *ngFor="let filter of timelineActivityFilters">
            <div class="filter-label">{{ filter.Label }}</div>
            <div class="text-right">
              <button (click)="toggleFilter(filter.Value)" class="reset-button">
                <div *ngIf="filter.IsEnabled">
                  <i class="fa fa-2x fa-toggle-on" ></i>
                </div>
                <div *ngIf="!filter.IsEnabled">
                  <i class="fa fa-2x fa-toggle-off"></i>
                </div>
              </button>
            </div>
          </li>
      </ul>
    </div>
    <div class="card-block">
      <div class="px-1">
        <div *ngIf="timelineActivitiesFiltered.length == 0" class="text-center">
          There is no recent activity to display at this time.
        </div>
        <ul class="timeline">
          <li *ngFor="let activity of timelineActivitiesFiltered">
            <div class="timeline-icon">
              <ng-container *ngIf="activity.Type === COMMUNITY_TYPE">
                <i *ngIf="activity.SubType !== 'Reply'" class="far fa-comment-alt"></i>
                <i *ngIf="activity.SubType === 'Reply'" class="fas fa-comments"></i>
              </ng-container>
              <i *ngIf="activity.Type === ACTIVITY_TYPE" class="fas fa-share"></i>
              <i *ngIf="activity.Type === RESOURCES_TYPE" class="fas fa-info"></i>
            </div>
            <div class="timeline-panel">
              <div class="timeline-header d-inline-flex  align-items-center">
                <img class="pf-avatar" [src]="activity.AvatarUrl">
                <h6 class="pl-2 mt-3">
                  <a href="{{ activity.PostedUrl }}" class="pr-2">{{ activity.PostedBy }}</a>
                </h6>
                <h6 class="mt-3" title="{{ activity.PostedTime }}">
                  {{ activity.ElapsedTime}}
                </h6>
              </div>
              <h6 class="timeline-source py-1"><span [innerHTML]="activity.Subject"></span></h6>
              <div *ngIf="activity.Body" class="timeline-body p-3"><span [innerHtml]="activity.Body | TruncateAfter:200 "></span></div>
            </div>
          </li>
        </ul>
      </div>
    </div>
    <div class="d-flex justify-content-center mb-3">
      <button type="button" class="btn btn-primary" disabled="disabled" *ngIf="timelineActivitiesFiltered.length == 0" >Show More</button>
      <button type="button" class="btn btn-primary" *ngIf="timelineActivitiesFiltered.length > 0" >Show More</button>
    </div>
  </div>
</div>
