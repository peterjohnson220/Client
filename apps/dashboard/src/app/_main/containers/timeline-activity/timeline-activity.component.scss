@import 'colors';

.timeline {
  list-style: none;
  padding: 0 0 1.25rem;
  position: relative;
  margin-top: -0.9375rem;
}

.timeline-body {
  background-color: $timeline-body-background-color;
  border-radius: 5%;
  border: thin;
  border-color: #f5f5f6;
}

.timeline-icon {
  background-color: #ffffff;
}

.timeline:before {
  top: 30px;
  bottom: 25px;
  position: absolute;
  content: " ";
  width: 0.1875rem;
  background-color: #eeeeee;
  left: 1.5625rem;
  margin-right: -1.5px
}

.timeline > li, .timeline > li > .timeline-panel {
  margin-bottom: 0.3125rem;
  position: relative
}

.timeline > li:after, .timeline > li:before {
  content: " ";
  display: table
}

.timeline > li:after {
  clear: both
}

.timeline > li > .timeline-panel {
  margin-left: 3.4375rem;
  float: left;
  top: 1.1875rem;
  padding: 0.25rem 0.625rem 0.5rem 0.9375rem;
}

.timeline > li > .timeline-icon {
  width: 2.25rem;
  height: 2.25rem;
  position: absolute;
  top: 26px;
  left: 7px;
  margin-right: -1.5625rem;
  background-color: #fff;
  z-index: 100;
  text-align: center;
  padding-top: 4px;
  border-radius: 100%;
  border: 0.125rem solid #d9d9d9 !important;
}

.card, .card-header {
  background-color: transparent;
}

.btn {
  width: 80%;
  background-color: $timeline-panel-button-color;
}
