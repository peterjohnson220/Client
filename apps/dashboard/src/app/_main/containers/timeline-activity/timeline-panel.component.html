<div class="card px-1 py-3">
  <div class="card-header d-flex align-items-center pl-1">
      <i class="fa fa-2x fa-commenting-o px-2" aria-hidden="true"></i>
      <h4>Recent Activity</h4>
  </div>
  <div class="card-block pt-3 pb-1">
    <pf-timeline-activity></pf-timeline-activity>
  </div>
  <div class="d-flex justify-content-center"><button type="button" class="btn btn-primary">Show More</button></div>
</div>

