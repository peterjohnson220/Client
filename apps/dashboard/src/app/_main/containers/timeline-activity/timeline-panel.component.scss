@import 'colors';
.card, .card-header {
  background-color: transparent;
}

.btn{
  width: 80%;
  background-color: $timeline-panel-button-color;
}
