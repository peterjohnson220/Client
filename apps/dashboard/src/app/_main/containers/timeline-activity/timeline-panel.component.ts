import { Component } from '@angular/core';

@Component({
  selector: 'pf-timeline-panel',
  templateUrl: './timeline-panel.component.html',
  styleUrls: ['./timeline-panel.component.scss']
})
export class TimelinePanelComponent {

}
