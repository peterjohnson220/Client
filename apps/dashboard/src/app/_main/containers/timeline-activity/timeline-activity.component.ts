import { Component, OnInit } from '@angular/core';
import { TimelineActivity } from '../../models/timeline-activity.model';

@Component({
  selector: 'pf-timeline-activity',
  templateUrl: './timeline-activity.component.html',
  styleUrls: ['./timeline-activity.component.css']
})
export class TimelineActivityComponent implements OnInit {
  private avatarUrl: string;
  private timelineActivities: TimelineActivity[];

  constructor( ) { }

  ngOnInit() {
      this.avatarUrl = 'https://f7021091349f6caaffd2-5b56effc7aa76a3323ddc3429496d092.ssl.cf5.rackcdn.com/avatars/default_user.png';

      this.timelineActivities = [];

      this.timelineActivities.push({
        Type: 'CommunityPost',
        SubType: 'Reply',
        PostedBy: 'John Clark',
        PostedTime: '3m ago',
        Subject: 'Replied To Mike Davidson\'s post',
        Body: 'Reply to Loriem Ipsum is simple a dummy text of the printing and typesetting industry.',
        AvatarUrl: '/assets/john.png'
      });

      this.timelineActivities.push({
        Type: 'CommunityPost',
        SubType: null,
        PostedBy: 'Mike Davidson',
        PostedTime: '5h ago',
        Subject: 'Posted to the <a href="#">Payfactors Community.</a>',
        Body: 'Loriem Ipsum is simple a dummy text of the printing and typesetting industry.',
        AvatarUrl: '/assets/adam.jpg'
      });

    this.timelineActivities.push({
      Type: 'ResourcesPost',
      SubType: null,
      PostedBy: 'Payfactors',
      PostedTime: '11h ago',
      Subject: 'Added new <a href="#">Release Notes.</a>',
      Body: null,
      AvatarUrl: '/assets/favicon.ico'
    });

    this.timelineActivities.push({
      Type: 'ActivityPost',
      SubType: null,
      PostedBy: 'Sue Jones',
      PostedTime: '2d ago',
      Subject: 'Shared <a href="#">Retail Jobs 2018</a> project with you.',
      Body: null,
      AvatarUrl: '/assets/sue.gif'
    });

    this.timelineActivities.push({
      Type: 'ResourcesPost',
      SubType: null,
      PostedBy: 'Payfactors',
      PostedTime: '7d ago',
      Subject: 'Added a new <a href="#">Video.</a>',
      Body: null,
      AvatarUrl: '/assets/favicon.ico'
    });


  }
}

