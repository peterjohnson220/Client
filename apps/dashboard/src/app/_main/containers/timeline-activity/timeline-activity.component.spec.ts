import { NO_ERRORS_SCHEMA } from '@angular/core';
import { TestBed, ComponentFixture } from '@angular/core/testing';
import { TimelineActivityComponent } from './timeline-activity.component';
import { getMockTimelineActivities } from '../../models/timeline-activity.model';

describe('timeline-activity', () => {
  let fixture: ComponentFixture<TimelineActivityComponent>;
  let instance: TimelineActivityComponent;

  // Configure Testing Module for before each test
  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [
        TimelineActivityComponent
      ],
      // Shallow Testing
      schemas: [ NO_ERRORS_SCHEMA ]
    });

    fixture = TestBed.createComponent(TimelineActivityComponent);
    instance = fixture.componentInstance;
  });

  it('should show timeline activities', () => {
    instance.timelineActivities = getMockTimelineActivities();
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });

  it('should show an icon with class fa-commenting-o when Type is CommunityPost and SubType is not Reply', () => {
    instance.timelineActivities = getMockTimelineActivities('CommunityPost');
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });

  it('should show an icon with class fa-comments when Type is CommunityPost and SubType is Reply', () => {
    instance.timelineActivities = getMockTimelineActivities('CommunityPost', 'Reply');
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });

  it('should show an icon with fa-share when Type is ActivityPost', () => {
    instance.timelineActivities = getMockTimelineActivities('ActivityPost');
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });

  it('should show an icon with fa-info when Type is ResourcesPost', () => {
    instance.timelineActivities = getMockTimelineActivities('ResourcesPost');
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });
});


