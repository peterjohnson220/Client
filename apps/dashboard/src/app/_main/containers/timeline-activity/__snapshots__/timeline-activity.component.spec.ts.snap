// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timeline-activity should show an icon with class fa-commenting-o when Type is CommunityPost and SubType is not Reply 1`] = `
<pf-timeline-activity
  timelineActivities={[Function Array]}
>
  <div
    class="card px-1 py-3"
  >
    
    
    <div
      class="card-header d-flex align-items-center pl-1"
    >
      
      
      <i
        aria-hidden="true"
        class="fa fa-2x fa-commenting-o px-2"
      />
      
      
      <h4>
        Recent Activity
      </h4>
      
    
    </div>
    
    
    <div
      class="card-block pt-3 pb-1"
    >
      
      
      <div
        class="px-1"
      >
        
        
        <ul
          class="timeline"
        >
          
          
          
          <li>
            
            
            <div
              class="timeline-icon"
            >
              
              
              
              
              
                
              
              <i
                class="fa fa-circle-thinner fa-commenting-o"
              />
              
                
              
              
              
              
              
              
              
              
              
              
            
            </div>
            
            
            <div
              class="timeline-panel"
            >
              
              
              <div
                class="timeline-header d-inline-flex  align-items-center"
              >
                
                
                <img
                  class="pf-avatar"
                  src="/assets/john.png"
                />
                
                
                <h6
                  class="pl-2 mt-3"
                >
                  
                  
                  <a
                    class="pr-2"
                    href="#"
                  >
                    John Clark
                  </a>
                  
                
                </h6>
                
                
                <h6
                  class="mt-3"
                >
                  3m ago
                </h6>
                
              
              </div>
              
              
              <h6
                class="timeline-source py-1"
              >
                <span>
                  Replied To Mike Davidson's post
                </span>
              </h6>
              
              
              
              <div
                class="timeline-body p-3"
              >
                <span>
                  Reply to Loriem Ipsum is simple a dummy text of the printing and typesetting industry.
                </span>
              </div>
              
            
            </div>
            
          
          </li>
          
        
        </ul>
        
      
      </div>
      
    
    </div>
    
    
    <div
      class="d-flex justify-content-center"
    >
      
      
      <button
        class="btn btn-primary"
        type="button"
      >
        Show More
      </button>
      
    
    </div>
    
  
  </div>
  
</pf-timeline-activity>
`;

exports[`timeline-activity should show an icon with class fa-comments when Type is CommunityPost and SubType is Reply 1`] = `
<pf-timeline-activity
  timelineActivities={[Function Array]}
>
  <div
    class="card px-1 py-3"
  >
    
    
    <div
      class="card-header d-flex align-items-center pl-1"
    >
      
      
      <i
        aria-hidden="true"
        class="fa fa-2x fa-commenting-o px-2"
      />
      
      
      <h4>
        Recent Activity
      </h4>
      
    
    </div>
    
    
    <div
      class="card-block pt-3 pb-1"
    >
      
      
      <div
        class="px-1"
      >
        
        
        <ul
          class="timeline"
        >
          
          
          
          <li>
            
            
            <div
              class="timeline-icon"
            >
              
              
              
              
              
                
              
              
                
              
              <i
                class="fa fa-circle-thinner fa-comments"
              />
              
              
              
              
              
              
              
              
              
            
            </div>
            
            
            <div
              class="timeline-panel"
            >
              
              
              <div
                class="timeline-header d-inline-flex  align-items-center"
              >
                
                
                <img
                  class="pf-avatar"
                  src="/assets/john.png"
                />
                
                
                <h6
                  class="pl-2 mt-3"
                >
                  
                  
                  <a
                    class="pr-2"
                    href="#"
                  >
                    John Clark
                  </a>
                  
                
                </h6>
                
                
                <h6
                  class="mt-3"
                >
                  3m ago
                </h6>
                
              
              </div>
              
              
              <h6
                class="timeline-source py-1"
              >
                <span>
                  Replied To Mike Davidson's post
                </span>
              </h6>
              
              
              
              <div
                class="timeline-body p-3"
              >
                <span>
                  Reply to Loriem Ipsum is simple a dummy text of the printing and typesetting industry.
                </span>
              </div>
              
            
            </div>
            
          
          </li>
          
        
        </ul>
        
      
      </div>
      
    
    </div>
    
    
    <div
      class="d-flex justify-content-center"
    >
      
      
      <button
        class="btn btn-primary"
        type="button"
      >
        Show More
      </button>
      
    
    </div>
    
  
  </div>
  
</pf-timeline-activity>
`;

exports[`timeline-activity should show an icon with fa-info when Type is ResourcesPost 1`] = `
<pf-timeline-activity
  timelineActivities={[Function Array]}
>
  <div
    class="card px-1 py-3"
  >
    
    
    <div
      class="card-header d-flex align-items-center pl-1"
    >
      
      
      <i
        aria-hidden="true"
        class="fa fa-2x fa-commenting-o px-2"
      />
      
      
      <h4>
        Recent Activity
      </h4>
      
    
    </div>
    
    
    <div
      class="card-block pt-3 pb-1"
    >
      
      
      <div
        class="px-1"
      >
        
        
        <ul
          class="timeline"
        >
          
          
          
          <li>
            
            
            <div
              class="timeline-icon"
            >
              
              
              
              
              
              
              
              
              
              <i
                class="fa fa-circle-thinner fa-info"
              />
              
            
            </div>
            
            
            <div
              class="timeline-panel"
            >
              
              
              <div
                class="timeline-header d-inline-flex  align-items-center"
              >
                
                
                <img
                  class="pf-avatar"
                  src="/assets/john.png"
                />
                
                
                <h6
                  class="pl-2 mt-3"
                >
                  
                  
                  <a
                    class="pr-2"
                    href="#"
                  >
                    John Clark
                  </a>
                  
                
                </h6>
                
                
                <h6
                  class="mt-3"
                >
                  3m ago
                </h6>
                
              
              </div>
              
              
              <h6
                class="timeline-source py-1"
              >
                <span>
                  Replied To Mike Davidson's post
                </span>
              </h6>
              
              
              
              <div
                class="timeline-body p-3"
              >
                <span>
                  Reply to Loriem Ipsum is simple a dummy text of the printing and typesetting industry.
                </span>
              </div>
              
            
            </div>
            
          
          </li>
          
        
        </ul>
        
      
      </div>
      
    
    </div>
    
    
    <div
      class="d-flex justify-content-center"
    >
      
      
      <button
        class="btn btn-primary"
        type="button"
      >
        Show More
      </button>
      
    
    </div>
    
  
  </div>
  
</pf-timeline-activity>
`;

exports[`timeline-activity should show an icon with fa-share when Type is ActivityPost 1`] = `
<pf-timeline-activity
  timelineActivities={[Function Array]}
>
  <div
    class="card px-1 py-3"
  >
    
    
    <div
      class="card-header d-flex align-items-center pl-1"
    >
      
      
      <i
        aria-hidden="true"
        class="fa fa-2x fa-commenting-o px-2"
      />
      
      
      <h4>
        Recent Activity
      </h4>
      
    
    </div>
    
    
    <div
      class="card-block pt-3 pb-1"
    >
      
      
      <div
        class="px-1"
      >
        
        
        <ul
          class="timeline"
        >
          
          
          
          <li>
            
            
            <div
              class="timeline-icon"
            >
              
              
              
              
              
              
              <i
                class="fa fa-circle-thinner fa-share"
              />
              
              
              
              
            
            </div>
            
            
            <div
              class="timeline-panel"
            >
              
              
              <div
                class="timeline-header d-inline-flex  align-items-center"
              >
                
                
                <img
                  class="pf-avatar"
                  src="/assets/john.png"
                />
                
                
                <h6
                  class="pl-2 mt-3"
                >
                  
                  
                  <a
                    class="pr-2"
                    href="#"
                  >
                    John Clark
                  </a>
                  
                
                </h6>
                
                
                <h6
                  class="mt-3"
                >
                  3m ago
                </h6>
                
              
              </div>
              
              
              <h6
                class="timeline-source py-1"
              >
                <span>
                  Replied To Mike Davidson's post
                </span>
              </h6>
              
              
              
              <div
                class="timeline-body p-3"
              >
                <span>
                  Reply to Loriem Ipsum is simple a dummy text of the printing and typesetting industry.
                </span>
              </div>
              
            
            </div>
            
          
          </li>
          
        
        </ul>
        
      
      </div>
      
    
    </div>
    
    
    <div
      class="d-flex justify-content-center"
    >
      
      
      <button
        class="btn btn-primary"
        type="button"
      >
        Show More
      </button>
      
    
    </div>
    
  
  </div>
  
</pf-timeline-activity>
`;

exports[`timeline-activity should show timeline activities 1`] = `
<pf-timeline-activity
  timelineActivities={[Function Array]}
>
  <div
    class="card px-1 py-3"
  >
    
    
    <div
      class="card-header d-flex align-items-center pl-1"
    >
      
      
      <i
        aria-hidden="true"
        class="fa fa-2x fa-commenting-o px-2"
      />
      
      
      <h4>
        Recent Activity
      </h4>
      
    
    </div>
    
    
    <div
      class="card-block pt-3 pb-1"
    >
      
      
      <div
        class="px-1"
      >
        
        
        <ul
          class="timeline"
        >
          
          
          
          <li>
            
            
            <div
              class="timeline-icon"
            >
              
              
              
              
              
              
              
              
              
              
            
            </div>
            
            
            <div
              class="timeline-panel"
            >
              
              
              <div
                class="timeline-header d-inline-flex  align-items-center"
              >
                
                
                <img
                  class="pf-avatar"
                  src="/assets/john.png"
                />
                
                
                <h6
                  class="pl-2 mt-3"
                >
                  
                  
                  <a
                    class="pr-2"
                    href="#"
                  >
                    John Clark
                  </a>
                  
                
                </h6>
                
                
                <h6
                  class="mt-3"
                >
                  3m ago
                </h6>
                
              
              </div>
              
              
              <h6
                class="timeline-source py-1"
              >
                <span>
                  Replied To Mike Davidson's post
                </span>
              </h6>
              
              
              
              <div
                class="timeline-body p-3"
              >
                <span>
                  Reply to Loriem Ipsum is simple a dummy text of the printing and typesetting industry.
                </span>
              </div>
              
            
            </div>
            
          
          </li>
          
        
        </ul>
        
      
      </div>
      
    
    </div>
    
    
    <div
      class="d-flex justify-content-center"
    >
      
      
      <button
        class="btn btn-primary"
        type="button"
      >
        Show More
      </button>
      
    
    </div>
    
  
  </div>
  
</pf-timeline-activity>
`;
